.gitignore

```diff
@@ -4,6 +4,8 @@
 [Bb]uild/
 [Bb]uilds/
 Assets/AssetStoreTools*
+Assets/ProCamera2D
```

Comment:
These shouldn't be necessary, we have them further down in the gitignore. 😄

Reply:
Haha, sorry, I didn't notice :)